### PR TITLE
Migration/textbracket

### PR DESCRIPTION
**Pull request by rvilarl**
TextBracket!

### CODE REVIEW

File: src/textbracket.ts [hunk 1]
```diff
@@ -11,29 +11,65 @@ import { Vex } from './vex';
...
+
+export interface TextBracketParams {
+  start: Note;
+  stop: Note;
+  text: string;
+  superscript: string;
+  position: number | string;
+}
+
+export interface TextBracketRenderOptions {
```
Comment (0xfe) [inline src/textbracket.ts:25]: This can be inlined.

Comment (rvilarl) [reply src/textbracket.ts:25]: done

**Comment by rvilarl** [comment]
Ready to merge

File: src/textbracket.ts [hunk 2]
```diff
@@ -11,29 +11,63 @@ import { Vex } from './vex';
...
+  protected readonly text: string;
+  protected readonly superscript: string;
+  protected readonly position: TextBracketPosition;
+
+  protected line: number;
+  protected start: Note;
+  protected stop: Note;
+  protected font: FontInfo;
+
+  static get Position(): typeof TextBracketPosition {
```
Comment (0xfe) [inline src/textbracket.ts:59]: This changes the interface -- can you add deprecated methods that log warnings `static get Positions()` and `static get PositionsString()`.

Comment (rvilarl) [reply src/textbracket.ts:64]: done

**Comment by rvilarl** [comment]
Ready to commit

**Comment by 0xfe** [comment]
Looks good.

**Pull request merged**